src/main/java/sysc4806/graduateAdmissions/model/Role.java

```diff
@@ -0,0 +1,60 @@
+package sysc4806.graduateAdmissions.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.util.*;
+
+/**
+ * This class specifies a role for the system's role-based access control.
+ * Each role has a unique name, and a set of privileges that specify what
+ * type of operations the role can perform.
+ *
+ * A User has one or more roles
+ *
+ * @author luke
+ */
+@Data
+@Builder
+public class Role {
+    //the primary key a role
+    @Id
+    private String roleName;
+    //the privileges for the role
+    @OneToMany(cascade = CascadeType.PERSIST)
+    @Builder.Default
+    private Set<Privilege> privileges = new HashSet<>();
```

Comment:
I see there is a @ OneToMany annotation, but this class isn't an @ Entity, so it isn't part of the JPA layer involved with the database. Is this meant to be an entity? If so, there should probably some JPA tests in their own test class too.

Reply:
yep, that's my bad. every model object we're making this week should have the entity annotation, so I forgot it in the Privilege too i think, I'll add those both